backend/src/WebServer/controllers/mail/MailController.py

```diff
@@ -0,0 +1,47 @@
+from src.util.LogFactory import LogFactory
+from src.WebServer.decorators.HTTPLogger import http_logger
+from src.WebServer.WebServerInit import WebServerInit
+from src.util.ErrorFactory import errorStackTrace
+from src.Mail.MailQ import MailQ
+import re
+
+from flask import Flask, jsonify, request
+
+flask_ref: Flask = WebServerInit.flask
+
+class MailController:
+
+  def __init__(self):
+    LogFactory.MAIN_LOG.info('Start MailController')
+
+  @staticmethod
+  @flask_ref.route('/mail', methods=['POST'])
+  @http_logger
+  def mail_api():
+    try:
+      LogFactory.MAIN_LOG.info("mail api")
+      email = request.args['email']
+      if emailIsValid(email):
+        mailQ = MailQ()
```

Comment:
you can actually grab the MailQ object from singletons: https://github.com/xxdunedainxx/lindsay-wildlife-treasure-hunt/blob/main/backend/src/Singletons.py#L9, just make sure https://github.com/xxdunedainxx/lindsay-wildlife-treasure-hunt/blob/main/backend/src/Singletons.py#L29 is set to true in configuration